src/util.js

```diff
@@ -40,36 +40,39 @@ module.exports = class Util {
         return ["◀", "⛔", "▶"];
     }
 
-    static async pagination(msg, author, contents, init = true, currPage = 0) {
-        if (init) for (const emoji of this.paginationEmojis) await msg.react(emoji);
-
-        const collector = msg.createReactionCollector((reaction, user) => {
-            return this.paginationEmojis.includes(reaction.emoji.name) && user.id === author.id;
-        }, {
+    static pagination(interaction, member, contents, currPage = 0) {
```

Comment:
why would we need to pass an interaction instead of a message, and member instead of author here?